homeassistant/components/fan/__init__.py

```diff
@@ -626,17 +642,15 @@ def state_attributes(self) -> dict:
 
         return data
 
-    @property
-    def supported_features(self) -> int:
```

Comment:
We cannot remove this, the fan platform doesn't allow `supported_features` to be a `None` value (as `Entity` allows). So it needs to stay here to re-type that fact.